libr/bin/pdb/pdb_downloader.c

```diff
@@ -115,3 +117,61 @@ void deinit_pdb_downloader(SPDBDownloader *pd) {
 	R_FREE(pd->opt);
 	pd->download = 0;
 }
+
+int r_bin_pdb_download(RCore* core, int isradjson, int* actions_done) {
+	int ret;
+	char *path;
+	SPDBDownloaderOpt opt;
+	SPDBDownloader pdb_downloader;
+	RBinInfo *info = r_bin_get_info (core->bin);
+	char *env_pdbserver = r_sys_getenv ("PDB_SERVER");
+	char *env_pdbextract = r_sys_getenv ("PDB_EXTRACT");
+	char *env_useragent = r_sys_getenv ("PDB_USER_AGENT");
+
+	if (!info || !info->debug_file_name) {
+		eprintf ("Can't find debug filename.\n");
+		return 1;
+	}
+
+	if (info->file) {
+		path = r_file_dirname (info->file);
+	} else {
+		path = strdup (".");
+	}
+
+	if (env_pdbserver && *env_pdbserver) {
```

Comment:
this is already accessible via env vars, r2 can modify the env with the % command, and you can add callbacks for those config vars to set the env vars, otherwise just make them accessible as parameters to the function or pass an struct pointer containing the settings of it